lib/trans/query_builder.ex

```diff
@@ -80,10 +80,9 @@ if Code.ensure_loaded?(Ecto.Query) do
     """
     defmacro translated(module, translatable, locale) do
       with field <- field(translatable) do
-        with {module_name, []} <- Module.eval_quoted(__CALLER__, module) do
-          validate_field(module_name, field)
-          generate_query(schema(translatable), module_name, field, locale)
-        end
+        module = Module.expand(module, __CALLER__)
```

Comment:
Wow! Thank you very much for the PR! ❤️ 

Unfortunately this line is causing the compilation to fail.
I think that you meant to invoke [`Macro.expand/2`][1]. Would that be correct?

[1]: https://hexdocs.pm/elixir/Macro.html#expand/2

Reply:
Yes, sorry. `Macro.expand` :)